.gitmodules

```diff
@@ -1,7 +1,7 @@
 [submodule "extern/qfr"]
 	path = extern/qfr
 	url = https://github.com/cda-tum/qfr.git
-    branch = main
+    branch = noiseComputeTable
```

Comment:
Before mergin, this should point back to qfr/main :)

Reply:
indeed.